corehq/apps/es/forms.py

```diff
@@ -54,3 +55,7 @@ def completed(gt=None, gte=None, lt=None, lte=None):
 
 def user_id(user_ids):
     return filters.term('form.meta.userID', list(user_ids))
+
+
+def doc_types(doc_types):
+    return filters.term('doc_type', map(lambda doc_type: doc_type.lower(), list(doc_types)))
```

Comment:
There's already a [`doc_type` filter](https://github.com/dimagi/commcare-hq/blob/cb3c4b0aa482d3d1bcf99b804ac8a5e4c4d26f86/corehq/apps/es/filters.py#L77-L79) on `HQESQuery` which accepts a list.  It doesn't coerce to lowercase though, so might have to push that to the client code or something if it's necessary.